pkg/registry/core/pod/storage/eviction.go

```diff
@@ -146,38 +148,59 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 			return err
 		}
 
-		if len(pdbs) > 1 {
-			rtStatus = &metav1.Status{
-				Status:  metav1.StatusFailure,
-				Message: "This pod has more than one PodDisruptionBudget, which the eviction subresource does not support.",
-				Code:    500,
-			}
-			return nil
-		}
 		if len(pdbs) == 0 {
 			return nil
 		}
 
-		pdb := &pdbs[0]
-		pdbName = pdb.Name
-		refresh := false
-		err = retry.RetryOnConflict(EvictionsRetry, func() error {
-			if refresh {
-				pdb, err = r.podDisruptionBudgetClient.PodDisruptionBudgets(pod.Namespace).Get(context.TODO(), pdbName, metav1.GetOptions{})
+		start := metav1.Time{Time: time.Now()}
```

Comment:
It wasn't clear to me from the discussion on the previous PR why this `start` timestamp is necessary. It might be good to have it briefly explained in a comment or something